recipes/dunovo/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+make
+mkdir $PREFIX/include/dunovo
+cp *.py *.so $PREFIX/include/dunovo
```

Comment:
why `include/`, do you have this directory hardcoded somewhere ?
libraries like .so files should go in `lib/`, executable scripts can go directly at `bin/` or you can put them at `share/` or `lib/` and make links if there's a lot of them.

the `include` dir should contain only header files.